### PR TITLE
Abb 7 78 90 168 199 203 BL

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records, corrections and updates to older records.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 1]
```diff
@@ -277,6 +277,32 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <locus from="228r"/>
                     <title type="complete" xml:lang="en" ref="LIT1895Martyr">Martyrdom of Eustathius, his wife and two children, under Marcianus (<date ref="ethiocal:Maskaram23">23rd of Maskaram</date>)</title>
                     <textLang mainLang="gez"/>
+
+
+                    <colophon xml:id="coloph1">
+                      <locus target="#241vb"></locus>
+                    <note xml:lang="en">The colophon refers obviously to <ref target="#ms_i1"/>,
+                    but it is located on the last page <ref target="#241vb"/> and should
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient691.xml:285]: locus, not ref

File: ParisBNF/abb/BNFabb168.xml [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+                           <layoutDesc>
+                              <layout columns="2" writtenLines="22 29">
+                                <note>The number of written lines increases to 28-29 starting from
+                                <ref target="#120v"/>.</note>
+                                <ab type="pricking">Pricking is visible.</ab>
+                                  <ab type="ruling">Ruling is poorly visible.</ab>
+                                <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                                <ab type="ruling">The upper line is written above the ruling.</ab>
+                                <note>The first leaf <ref target="#i"/> is neither ruled nor pricked.</note>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb168.xml:119]: locus

File: ParisBNF/abb/BNFabb199.xml [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+                        <origin>
+                     <origDate notBefore="1770" notAfter="1849">Late 18th/first half of the 19th century</origDate>.
+                        </origin>
+                        <provenance>The manuscript was acquired by
+                          <persName ref="PRS9683Urbino">Giusto da Urbino</persName> at
+                          <placeName ref="LOC5843Tadbab">Tadbāba Māryām</placeName> from some anonymous sellers called
+                           disciples of <persName ref="PRS13840Yamesrac">ʾabbā Yamǝsrāč</persName>, on <date>1 October 1849 CE</date>
+                           (according to <ref target="#a2">Addition 2</ref>).
+                           It is one of the manuscripts that <persName ref="PRS9683Urbino" role="owner"/>
+                           bequethed to <persName ref="PRS1071dAbbadi"/>.
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb199.xml:224]: bequeathed

File: ParisBNF/abb/BNFabb203.xml [hunk 4]
```diff
@@ -36,31 +36,33 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
 
-                            <msItem xml:id="ms_i1.1">
-                              <title type="complete" ref="LIT1546Genesi" xml:lang="gez"/>
-                            <incipit>
+                            <incipit xml:lang="gez">
                             <locus target="#1ra"/><hi rend="rubric">ኦሪት᎓ ዘልደት᎓
                             ምዕራፍ፩᎓ በቀዳሚ᎓ ገብረ᎓ እግዚአብሔር᎓</hi> ሰማየ᎓ ወምድረ። ወምድርሰ᎓ ሀለወት᎓ ዕራቃ።
                           </incipit>
-                          <explicit>
+                          <explicit xml:lang="">
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb203.xml:44]: language value missing

File: ParisBNF/abb/BNFabb203.xml [hunk 5]
```diff
@@ -322,10 +336,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                      ዘበትርጓሜሁ᎓ ኮከብ᎓</q>, <ref target="#ms_i1.2"/>.</note>
                   </origin>
-                  <provenance><persName ref="PRS8972Stella">Giovanni Stella</persName>
-                sold the manuscript to <persName ref="PRS9683Urbino">
-                Giusto da Urbino</persName>.</provenance>
-                  <acquisition/>
+                  <provenance><persName ref="PRS8972Stella" role="owner">Giovanni Stella</persName>
+                sold the manuscript to <persName ref="PRS9683Urbino" role="owner">
+                Giusto da Urbino</persName>. It is one of the manuscripts that <persName ref="PRS9683Urbino" role="bequeather"/>
+                bequethed to <persName ref="PRS1071dAbbadi" role="owner"/>.
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb203.xml:342]: bequeathed

File: ParisBNF/abb/BNFabb90.xml [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+                                      <height>210</height>
+                                      <width>180</width>
+                                    </dimensions>
+                                </extent>
+
+                                <foliation>Foliation in pencil in Eastern Arabic numerals,
+                                  in the upper right corner of the folia.
+                                  The first unfoliated (paper?) leaf is a part of the
+                                  binding. The last parchment folium, unfoliated, is
+                                  assigned the number <locus target="#207">115</locus>.</foliation>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb90.xml:84]: 207 or 115?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient691.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient691.xml:285]: ok

File: ParisBNF/abb/BNFabb199.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb199.xml:224]: ok

File: ParisBNF/abb/BNFabb203.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb203.xml:44]: ok

File: ParisBNF/abb/BNFabb90.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb90.xml:84]: 115

**Review by thea-m — APPROVED** [review]

**Pull request merged**